src/EnergyPlus/DirectAirManager.cc

```diff
@@ -625,7 +634,7 @@ namespace DirectAirManager {
 		MaxAirVolFlowRateUser = 0.0;
 		SizingDesRunThisZone = false;
 
-		if ( CurZoneEqNum > 0 ) {
+		if ( ( CurZoneEqNum > 0 ) && ( CurTermUnitSizingNum > 0 ) ) {
```

Comment:
So what happens here if CurTermUnitSizingNum = 0? Sizing gets skipped? I assume it would never be 0 and this is just a safety check. CurZoneEqNum is no longer used, does it still need to be checked now? Better safe than sorry I guess.